src/main/java/org/gridsuite/directory/server/repository/DirectoryElementRepository.java

```diff
@@ -51,6 +51,10 @@ public interface DirectoryElementRepository extends JpaRepository<DirectoryEleme
     @Query("SELECT name FROM DirectoryElementEntity WHERE parentId=:parentId AND type=:type AND name like :name% and stashed = false ")
     List<String> getNameByTypeAndParentIdAndNameStartWith(String type, UUID parentId, String name);
 
+    //We also count when type = study because every study is linked to a case that is not visible in DB
+    @Query("SELECT count(*) FROM DirectoryElementEntity WHERE owner=:owner AND type='CASE' OR type='STUDY'")
```

Comment:
You forgot the parentheses in the query. 
`@Query("SELECT count(*) FROM DirectoryElementEntity WHERE owner=:owner AND (type='CASE' OR type='STUDY')")`